guides/doc-Release_notes/master.adoc

```diff
@@ -0,0 +1,27 @@
+include::common/attributes.adoc[]
+
+:numbered:
+:toc:
+:toc-placement: left
+
+ifdef::katello[]
+= Foreman {ProjectVersion} and Katello {KatelloVersion} Release Notes
+endif::[]
+
+include::topics/foreman.adoc[leveloffset+=1]
```

Comment:
```suggestion
include::topics/foreman.adoc[leveloffset=+1]
```

Please change this throughout the file.

Reply:
This is probably why I had trouble with the headers.